lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java

```diff
@@ -137,10 +138,39 @@ private void drawTextGlyphs(
     float fontScale = (float) documentData.size / 100f;
     float parentScale = Utils.getScale(parentMatrix);
 
-    float totalTextWidth = getTotalTextWidthForGlyphs(documentData, font, fontScale, parentScale);
-    applyJustification(documentData.justification, canvas, totalTextWidth);
-
     String text = documentData.text;
+
+    // Line height
+    float lineHeight = (float) documentData.lineHeight * Utils.dpScale();
+
+    // Split full text in multiple lines
+    List<String> textLines = getTextLines(text);
+    int textLineCount = textLines.size();
+    for (int l = 0; l < textLineCount; l++) {
+
+      String textLine = textLines.get(l);
+      float textLineWidth = getTextLineWidthForGlyphs(textLine, font, fontScale, parentScale);
+
+			canvas.save();
```

Comment:
There is something funky going on with the formatting here.

Reply:
Thanks for spotting this formatting issue from my side.